app/src/main/res/values/strings.xml

```diff
@@ -341,6 +341,7 @@
   <!-- OptionsActivity -->
   <string name="app_language_activity_title">App Language</string>
   <string name="audio_language_activity_title">Default Audio Language</string>
+  <string name="reading_text_size_activity_title">Reading Text Size</string>
   <string name="title_reading_text_size">Reading Text Size</string>
```

Comment:
Not using this as it is part of `fragment` and also cannot rename this as as per our convention the labels are named as `<name>_activity_title`.